docs/learn/learn-jam-faq.md

```diff
@@ -0,0 +1,163 @@
+to get validated by the validators, etc. JAM will offer the possibility of running smart contracts
+on the same level as parachains. This means a potential service can be written that enables people
+to run Solidity-based smart contracts directly on top of JAM without running on any parachain. The
+other more future-oriented development will be CorePlay. CorePlay will be an actor-based framework.
+The idea is to support long-running tasks/actors on top of JAM. Long-running here means that
+programs will continue running intermittently (can pause and resume). This provides quite a lot of
+simplifications for the developer when writing contracts. Ultimately, it should be much simpler to
+write a program that can run on top of a (decentralized) blockchain than it is now.
+
+## How can developers get involved with JAM?
+
+Developers interested in JAM can explore and participate once the
+[RFC](../general/start-building.md#requests-for-comment-rfcs) is out. Additionally, they can stay
+updated on announcements and events related to JAM and its integration with other technologies via
+Parity’s communication channels.
+
+## What are some potential applications of JAM?
+
+JAM's flexible and efficient computational model opens up possibilities for various applications,
+but it is too early to jump into specific use cases just yet.
+
+## Is there a roadmap for JAM's development?
+
+While specific timelines may vary, the development of JAM typically involves ongoing research,
+testing, and iteration. The RFC will be published as a first step, where the community gives
+feedback and iterates on it. The timeframe that JAM is expected to be production-ready is 20-60
+months from its presentation at Token2049 in Dubai (April 2024), not including the service layer.
+
+## What about Polkadot? Will there be a hard fork?
+
+JAM isn’t a successor to Polkadot but a potential evolution of the Polkadot Relay Chain. It can only
+proceed if it passes through Polkadot’s decentralized governance system, winning approval from token
+holders. The current parachain-centered logic would continue as a service on top of the new JAM
+architecture. Part of the proposal will include tooling and hard-coded compatibility guarantees. The
+Relay Chain will upgrade, and parachains can then enjoy a much richer environment, synchronous
+compatibility, and excellent scaling capabilities better than current L2s but built inside the L0/L1
+layer.
+
+## Do parachains need to rewrite their code? How much work is it to build?
+
+Parachains will be one of the “products” that can run on JAM. JAM will only remove the enshrined
+parachain consensus and replace it with a model supporting different services. For the first years
+after launch, it will still be faster to develop parachains, as the necessary tooling would need to
+be developed. For the migration, full compatibility guarantees will be written into the code.
```

Comment:
Clarity needed here-

For the first years
after launch, it will still be faster to develop parachains **(when compared to developing what?)**, as the necessary tooling **(for what?)** would need to
be developed.

Reply:
compared to building services, as the necessary tooling for those would need to be developed.